app/src/main/java/com/example/series_collector/data/Series.kt

```diff
@@ -4,22 +4,25 @@ import androidx.room.ColumnInfo
 import androidx.room.Entity
 import androidx.room.PrimaryKey
 import com.google.firebase.firestore.DocumentId
-import java.util.Calendar
+import com.google.firebase.firestore.ServerTimestamp
+import java.util.*
 
 @Entity(tableName = "Series")
 data class Series(
     @PrimaryKey
     @DocumentId
     @ColumnInfo(name = "id")
-    val SeriesId: String,
+    val SeriesId: String = "",
```

Comment:
```suggestion
    val id: String = "",
```
series class의 property이므로 id만 명시하거나
seriesId로 작성하는게 적절해보입니다.